bliss/conf/base_config.yaml

```diff
@@ -142,35 +165,16 @@ predict:
 surveys:
     sdss:
         _target_: bliss.surveys.sdss.SloanDigitalSkySurvey
-        sdss_dir: ${paths.sdss}
-        sdss_fields:
+        dir_path: ${paths.sdss}
+        fields:  # TODO: better arbitary name for fields/bricks?
             - run: 94
               camcol: 1
               fields: [12]
         psf_config:
             pixel_scale: 0.396
             psf_slen: 25
         reference_band: 2 # SDSS r-band
-        prior_config:
-            max_bands: 5
-            n_tiles_h: 20
-            n_tiles_w: 20
-            tile_slen: 4
-            batch_size: 64
-            max_sources: 1
-            mean_sources: 0.2
-            min_sources: 0
-            prob_galaxy: 0.72
-            star_flux_min: 622
-            star_flux_max: 1e6
-            star_flux_alpha: 0.43
-            galaxy_flux_min: 622.0
-            galaxy_flux_max: 1e6
-            galaxy_alpha: 0.47
-            galaxy_a_concentration: 0.39330758068481686
-            galaxy_a_loc: 0.8371888967872619
-            galaxy_a_scale: 4.432725319432478
-            galaxy_a_bd_ratio: 2.0
+        max_bands: 5
```

Comment:
Will you change `max_bands` to `n_bands` (because it's just a constant property of the survey) and move `reference_band` to the prior (because it's not really a property of the survey--it's our model)?